### ISSUE TITLE
I cannot compile

### ISSUE THREAD

**Issue by Selim042** [CoderPuppy/lua-nn#1]
```
Installing https://luarocks.org/nn-1.0.3-1.src.rock...
Using https://luarocks.org/nn-1.0.3-1.src.rock... switching to 'build' mode
gcc -O2 -fPIC -I/usr/include/lua5.2 -c lnn.c -o lnn.o
lnn.c: In function ‘luaopen_nn’:
lnn.c:35:3: error: ‘for’ loop initial declarations are only allowed in C99 or C11 mode
   for(int n = 0; (name = nn_symbol(n, &value)) != NULL; n++) {
   ^
lnn.c:35:3: note: use option -std=c99, -std=gnu99, -std=c11 or -std=gnu11 to compile your code

Error: Build error: Failed compiling object lnn.o
```

**Comment by CoderPuppy** [comment]
Should be fixed by 099cc47 (released as 1.0.4-1)

**Comment by Selim042** [comment]
Confirmed.
